tests/plots/test_bar.py

```
@@ -0,0 +1,321 @@
+from itertools import cycle
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import pytest
+from matplotlib.axes import Axes
+
+from pyretailscience.plots import bar
+from pyretailscience.style import graph_utils as gu
+
+
+@pytest.fixture()
```

Ruff

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)